package.json

```diff
@@ -15,30 +14,31 @@
 	"devDependencies": {
 		"@esbuild-plugins/node-globals-polyfill": "^0.1.1",
 		"@rollup/plugin-inject": "^4.0.4",
-		"@sveltejs/adapter-auto": "next",
-		"@sveltejs/kit": "^1.0.0-next.400",
+		"@sveltejs/adapter-auto": "1.0.0-next.80",
+		"@sveltejs/kit": "1.0.0-next.503",
 		"@tailwindcss/typography": "^0.5.2",
 		"@typescript-eslint/eslint-plugin": "^5.27.0",
 		"@typescript-eslint/parser": "^5.27.0",
 		"autoprefixer": "^10.4.4",
-		"eslint": "^8.16.0",
-		"eslint-config-prettier": "^8.3.0",
+		"eslint": "^8.24.0",
+		"eslint-config-prettier": "^8.5.0",
 		"eslint-plugin-svelte3": "^4.0.0",
 		"postcss": "^8.4.12",
 		"postcss-load-config": "^3.1.4",
-		"prettier": "^2.6.2",
-		"prettier-plugin-svelte": "^2.7.0",
+		"prettier": "^2.7.1",
+		"prettier-plugin-tailwindcss": "^0.1.13",
```

Comment:
@aeither what is the reason to replace that?

Reply:
@aeither you have resolved this conversation, but I didn't get any answer about this. Please, could you me explain why this should be replaced? Thanks!!!